roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,3 @@
 dependencies:
   - pip
+  - logrotate
```

Comment:
Why is it better to have this extra abstraction, rather than just put the dict in the meta file?